studio/pages/project/[ref]/functions/[id]/details.tsx

```diff
@@ -97,13 +99,9 @@ const PageLayout: NextPageWithLayout = () => {
   ]
 
   // Get the API service
-  const { data: settings } = useProjectSettingsQuery({ projectRef: ref as string })
-  const apiService = settings?.autoApiService
-  const anonKey = apiService?.service_api_keys.find((x: any) => x.name === 'anon key')
-    ? apiService.defaultApiKey
-    : undefined
```

Comment:
We probably need to keep the current way of finding `anonKey` until we update the api. Currently this might be a bogus value when `anon` isn't in the list of `service_api_keys` https://github.com/supabase/infrastructure/blob/1e99f3eb66d2e6b1a42f26e0034c06377aaaa4c1/api/src/routes/platform/props/project/ref/api.controller.ts#L68

Reply:
Updated!